tests/neo-vm.Tests/Tests/OpCodes/BitwiseLogic/EQUAL.json

```diff
@@ -338,16 +338,12 @@
       "script": [
         "PUSH0",
         "NEWARRAY",
-        "TOALTSTACK",
-        "DUPFROMALTSTACK",
-        "DUPFROMALTSTACK",
+        "DUP",
```

Comment:
Good thing we use names in these tests... instead of opcodes in bytes :joy: @shargon